src/components/DataTable/index.jsx

```diff
@@ -38,7 +44,8 @@ class DataTable extends Component {
   };
 
   renderTable = formattedColumns => {
-    const { data, pageSize, onRowClick } = this.props;
+    const { data, pageSize: defaultPageSize, onRowClick } = this.props;
+    const { page: currentPage, pageSize: currentPageSize } = this.state;
```

Comment:
I'm not sure it's worth renaming these. State always represents the "current" so I don't see the value in the rename and it adds more variable names to keep track of. Other than that, sweet 👍

Reply:
Good point. I guess I got carried away as I had to at least change the pageSize prop name. The state names have been put back.